appveyor.yml

```
@@ -25,6 +25,7 @@ branches:
   only:
     - master
     - nightly
+    - 0.19.x
```

More generic for future needs:
```suggestion
    - /\d+\.\d+\.[\dx]+/
```

---

Applied.